src/tactic/linarith/frontend.lean

```diff
@@ -360,4 +371,4 @@ add_tactic_doc
 { name       := "nlinarith",
   category   := doc_category.tactic,
   decl_names := [`tactic.interactive.nlinarith],
-  tags       := ["arithmetic", "decision procedure", "finishing"] }
+  tags       := ["arithmetic", "finishing"] }
```

Comment:
I think it can still be considered a decision procedure, right?

Reply:
I guess, kind of. Everything's a decision procedure if you define the decision problem to be exactly what it decides. There's probably a better way to characterize the fragment of nonlinear arithmetic problems that `nlinarith` solves than just "by implementation." Not sure how natural it is.

But maybe it's better to leave the tag even if it's a partial lie, for the sake of discoverability. We don't have so many decision procedures. Commit this if you agree.

```suggestion
  tags       := ["arithmetic", "decision procedure", "finishing"] }
```

Reply:
I would say it is a decision procedure for linear arithmetic, and it also does decently well on other stuff.